pydc_client.py

```diff
@@ -381,7 +381,7 @@ def server_handler(self,data,info,args): # Interacts with the DC, responding to
 						if nick=="": continue
 						self._nicklist[nick]["bot"] = (True if nick in bots else False)
 				elif x[0]=="$MyINFO":
-					nick,desc,conn,flag,email,share = re.findall("^\$MyINFO \$ALL ([^ ]*) ([^\$]*)\$ \$([^\$]*)([^\$])\$([^\$]*)\$([^\$]*)\$$",data)[0]
+					nick,desc,conn,flag,email,share = re.findall("^\$MyINFO \$ALL ([^ ]*) ([^\$]*)\$.\$([^\$]*)([^\$])\$([^\$]*)\$([^\$]*)\$",data)[0]
```

Comment:
There are clients that that don't put a space where I put the `.`. Either there is no space or there is a random alphabet. While testing the bot was crashing for a particular string, which I am not being able to find now.